editor/test/integration/save_data_test.clj

```diff
@@ -628,7 +629,20 @@
                  :name "normal"
                  :wrap-u :wrap-mode-clamp-to-edge
                  :wrap-v :wrap-mode-clamp-to-edge}]
-               (g/node-value legacy-textures-material :samplers)))))))
+               (g/node-value legacy-textures-material :samplers)))))
+
+    (testing "model"
+      (let [legacy-material-and-textures-model (project/get-resource-node project "/silently_migrated/legacy_material_and_textures.model")
+            material-resource (workspace/find-resource workspace "/builtins/materials/model.material")
+            tex0-resource (workspace/find-resource workspace "/referenced/images/red.png")
+            tex1-resource (workspace/find-resource workspace "/referenced/images/green.png")]
+        (is (= [{:material material-resource
+                 :name "default"
+                 :textures [{:sampler "tex0"
+                             :texture tex0-resource}
+                            {:sampler "tex1"
+                             :texture tex1-resource}]}]
+               (g/node-value legacy-material-and-textures-model :materials)))))))
```

Comment:
Here I'm adding a simple test to the `silent-migrations-test` to verify that the previous model format loads correctly.